src/main/java/com/efficio/etl/web/controller/angular/AngularMapOntologyController.java

```diff
@@ -318,4 +326,19 @@ public FileUploadForm getForm() {
 		return new FileUploadForm();
 	}
 
+	public void validateTrialInstanceValue(final List<MeasurementVariable> conditions, final Map<String, List<Message>> errorMessages) {
+		for (final MeasurementVariable varCondition : conditions) {
+			if (varCondition.getTermId() == TermId.TRIAL_INSTANCE_FACTOR.getId()) {
+				final Optional<Message> message = Util.validateVariableValues(varCondition, varCondition.getValue());
+				if (message.isPresent()) {
+					if (errorMessages.containsKey(Constants.INVALID_TRIAL)) {
```

Comment:
if-else can be simplified as:
'errorMessages.putIfAbsent(Constants.INVALID_TRIAL, new ArrayList<>());
					errorMessages.get(Constants.INVALID_TRIAL).add(message.get());'